azula/plugins/adm/__init__.py

```diff
@@ -187,7 +187,7 @@ def load_model(key: str, **kwargs) -> ImprovedDenoiser:
         state = torch.load(
             f=cached_download(url=url),
             **kwargs,
-        )
+        weights_only=True)
```

Comment:
`weights_only=True` is already set by default in the `kwargs` dictionary.

Reply:
I see now. Then it's already fine, just hard to see with static analysis.